sh_scrapy/crawl.py

```diff
@@ -2,6 +2,7 @@
 # --------------------- DO NOT ADD IMPORTS HERE -------------------------
 # Add them below so that any import errors are caught and sent to sentry
 # -----------------------------------------------------------------------
+from __future__ import print_function
```

Comment:
I don't see real reasons to enforce `absolute_imports` everywhere looking to the [comment](https://github.com/scrapinghub/python-hubstorage/pull/62#issuecomment-216039081) but let me know if I'm missing something.